CI: invalid makefile target goimports

## Bug Report

See https://github.com/pingcap/chaos-mesh/pull/565/checks?check_run_id=719195154.

We are running `make goimports` here but actually we only have `groupimports` in the makefile https://github.com/pingcap/chaos-mesh/blob/5ead37a9e03827e0e1c9e8a40c6e81210dcd8e95/Makefile#L150

```
ensuring kustomize
make: *** No rule to make target 'goimports'.  Stop.
0
GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS="" GOARCH="" go get golang.org/x/tools/cmd/goimports@v0.0.0-20200309202150-20ab64c0d93f
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -37,7 +37,7 @@ jobs:
           export PATH=$PATH:$GOPATH/bin
           make ensure-kubebuilder || echo 0
           make ensure-kustomize || echo 0
-          make goimports || echo 0
+          make groupimports || echo 0
           #use sh function
 
           if [[ "$job" == "verify" ]]; then

